crates/bevy_derive/src/bevy_main.rs

```diff
@@ -10,10 +10,12 @@ pub fn bevy_main(_attr: TokenStream, item: TokenStream) -> TokenStream {
     );
 
     TokenStream::from(quote! {
-        // use ndk-glue macro to create an activity: https://github.com/rust-windowing/android-ndk-rs/tree/master/ndk-macro
+        
         #[cfg(target_os = "android")]
-        #[cfg_attr(target_os = "android", bevy::ndk_glue::main(backtrace = "on", ndk_glue = "bevy::ndk_glue"))]
-        fn android_main() {
+        fn android_main( android_app: bevy::android::AndroidApp) {
```

Comment:
The `android_main` entry point also needs to be `#[no_mangle]` since standalone native android applications are built as a cdylib and the runtime is going to use `LoadLibrary` + `dlsym()` to look up this entry point.

Reply:
Its there.

Reply:
aah, right I see there's another patch that adds it